### PR TITLE
Update styles.css

### PR DESCRIPTION
**Pull request by Shivam-Shandilya1**
## What is the change?
Added the .btnFrame named class and added the btn into that class having similar css styles

## Related issue?
close: #576

## How was it tested?
I have tested it by running it in my localhost:3000 and inspecting it by the google developer tool.

## Checklist:
Before you create this PR, confirm all the requirements listed below by checking the checkboxes `[x]`:

- [x] Have you followed the [Contribution Guidelines](https://github.com/ALPHAVIO/BlogSite/blob/master/CONTRIBUTING.md) while contributing.
- [x] Have you checked there aren't other open [Pull Requests](https://github.com/ALPHAVIO/BlogSite/pulls) for the same update/change?
- [x] Have you made corresponding changes to the documentation?
- [x] Your submission doesn't break any existing feature.
- [x] Have you tested the code before submission?

## Screenshots or Video:

https://user-images.githubusercontent.com/72553479/113530921-f0cd4a80-95e4-11eb-9e6d-dfd840811348.mp4

### CODE REVIEW

**Comment by Shivam-Shandilya1** [comment]
Please review my pull request.

File: views/partials/header.ejs [hunk 1]
```diff
@@ -51,7 +51,7 @@
                 </li>
               <% }else{ %>
                   <li id="log-in">
-                    <a href="/log-in" class="btn btn-info btn-sm text-white" role="button">Log in</a>
+                    <a href="/log-in" class="btn btn-info btn-sm text-white" role="button" style="margin-right: 5px;">Log in</a>
```
Comment (Pranav016) [inline views/partials/header.ejs:54]: This should be in a separate PR since this is for a different issue. Make separate branches for them..

**Review by Pranav016 — CHANGES_REQUESTED** [review]
* change base branch to master
* re write commit messages

**Comment by Pranav016** [comment]
closing this PR for now, make separate PRs solving separate issues..
